en-US/Docs/IoTCoreShell.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: docs
+title: IoT Core Shell
+description: An overview of how to think about and use the IoT Core Shell
+keyword: windows iot, shell, foreground, background, application
+permalink: /en-US/Docs/IoTCoreShell.htm
+lang: en-US
+---
+
+# IoT Core Shell Overview
+
+This document covers the IoT Core Shell, foreground and background applications, and how to navigate between these applications on your device.
+
+## IoT Shell, Foreground, and Background Apps
+
+The **IoT Shell** (iotshell.exe) is the part of IoT Core responsible for launching UI, bringing up foreground applications and continuous background applications on your device, sequencing operations like provisioning and OOBE events, monitoring applications for servicing events, and orchestrating certain other services. It's launched by the Shell Infrastructure Host service (sihost.exe) through a registry-configured IoT plugin (IoTShellExt.dll). Both the IoT Shell and the Host service are system critical, which means IoT Core will bluescreen if either one of these processes fails outside of a clean shutdown / reset.

```

Comment:
This is a super internal implementation detail that should be reworded in a simpler way.
E.g. IoT Core device does run a shell that will make sure registered startup apps are launched. It supports Headed and Headless modes. In case of Headed it will launch a single registered Headed startup app that will be full screen.